### PR TITLE
Improved READMEs

### PR DESCRIPTION
**Pull request by isaac-vb**
Now includes a basic description and also screenshots from the Tableau dashboard.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -11,7 +11,7 @@ Despite reporting on the same topics, different media organisations can have wil
...
+This project extracts UK news articles from different RSS feeds and finds related articles from other news feeds as well as comments about that story/topic on Reddit. Sentiment analysis is then applied to these news articles and social media comments and scores them depending on how positive/negative they are (-1 being the most negative, 1 being the most positive). The articles then have their topic keywords extracted using ChatGPT, this allows for articles/social media posts talking about the same news story/topic can be linked together. All of this information is then loaded onto an RDS which is running PostgreSQL. From the RDS, we visualise the wide variety of information we have gathered via a Tableau dashboard and also send out an email report with a PDF attached showing the most popular stories of the day amongst other key insights.
```
Comment (ramiz76) [inline README.md:14]: As we are only using BBC and DailyMail, it might be worth mentioning these two sources here.

Comment (ramiz76) [reply README.md:14]: The phrase "this allows for articles/social media posts talking about the same news story/topic can be linked together." may need to be adjusted as it is not grammatically correct

File: dashboardwordcloud.png
Comment (ramiz76) [inline dashboardwordcloud.png:1]: The top 75 topics by story count are presented within this word cloud where users are able to select specified topics to find key insights and a list of story titles that are used for the analysis.

**Review by rk145lr — APPROVED** [review]
Changes look good

File: maindashboard.png
Comment (ramiz76) [inline maindashboard.png:1]: The topics with the 10 highest/10 lowest sentiment scores are displayed using a diverging bar chart

**Pull request merged**